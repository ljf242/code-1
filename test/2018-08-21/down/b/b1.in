20 20
6 8 2 10 4 3 20 1 2 9 2 9 5 2 4 1 7 4 12 11
