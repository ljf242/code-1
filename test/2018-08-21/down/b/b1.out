20.000000000000 20 1 0
